plugins/inputs/lotus/lotus.go

```diff
@@ -132,6 +136,15 @@ func (l *lotus) Start(acc telegraf.Accumulator) error {
 	}
 	l.api = nodeAPI
 
+	nodePeerID, err := l.api.ID(context.Background())
+	if err != nil {
+		return err
+	}
+
+	if err := recordLotusInfoPoints(context.Background(), l.api, acc, time.Now(), nodePeerID); err != nil {
```

Comment:
Push `time.Now()` into the function and remove as argument, along with nodePeerID.